templates/concept.html

```diff
@@ -44,7 +44,7 @@ <h3 class="topic-card__myft-title">
 						<a class="topic-card__concept-empty-link" data-trackable="empty-link" href={{url}}>
 							See more on
 							{{#if nTopicCardEmptyTopic}}
-								the topic {{nTopicCardEmptyTopic}}
+								the topic "{{#if prefLabel}}{{prefLabel}}{{else}}{{name}}{{/if}}"
```

Comment:
is the quotation `"` needed? how is this read in the screen readers?

Reply:
I put quotes because of the suggestion of DAC that seemed to include them. (I did not try a screen reader)
<img width="920" alt="Screenshot 2022-11-16 at 10 19 42" src="https://user-images.githubusercontent.com/107469842/202154518-1d84a42f-29ea-4984-a997-805f5d4e03a3.png">

Reply:
ok so it is saying "Quote" out loud so Ill double check with DAC

Reply:
Ok so DAC bosses said no quote, it was just to say "insert heading" so I am removing it!